kore/src/Kore/Step/Step.hs

```diff
@@ -535,17 +585,16 @@ applyRulesParallel
     => UnificationProcedure
     -> [RulePattern variable]
     -- ^ Rewrite rules
-    -> Pattern variable
+    -> Pattern (Target variable)
     -- ^ Configuration being rewritten
     -> unifier (Results variable)
 applyRulesParallel
     unificationProcedure
     -- Wrap the rule and configuration so that unification prefers to substitute
     -- axiom variables.
     (map toAxiomVariables -> rules)
-    (toConfigurationVariables -> initial)
-  =
-    unifyRules unificationProcedure initial rules
+    initial
```

Comment:
Why did we remove `toConfigurationVariables` here?

Reply:
I think Traian made this change and the one from `applyRulesSequence`. @traiansf , could you help us? I couldn't follow you at this part

Reply:
But I think that was moved to `applyRewriteRulesParallel`
https://github.com/kframework/kore/pull/1026/files/c5216f71d35ce543a37d6eba5c3d8384e546522c#diff-595068f6f4dcaabbeb534249f4fd4c12R620